src/YesSql.Core/Sql/SqlBuilder.cs

```diff
@@ -158,9 +160,12 @@ public virtual string FormatColumn(string table, string column, bool isAlias = f
             if (!isAlias)
             {
                 table = _tablePrefix + table;
+                return _dialect.SchemaNameQuotedPrefix() + _dialect.QuoteForTableName(table) + "." + column;
+            }
+            else
+            {
+                return _dialect.QuoteForTableName(table) + "." + column;
```

Comment:
No schema here?

Reply:
It's an Alias so it doesn't take a Schema prefix. The real table names take a Schema name but once it is aliased it's not called with a schema.